### ISSUE TITLE
General settings will not save, no translations

### ISSUE THREAD

**Issue by chris-barklem** [solidusio/solidus#1176]
New build with latest updates will not save general settings such as site name, seo title, meta keywords, meta description, site url and mail from address.

With localization settings configured in config file and visible in: /admin/general_settings/edit but no translations are seen in taxonomy. 

Using postgres, environment development.

Gemfile:

````
source 'https://rubygems.org'

ruby '2.3.1'
# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '4.2.6'
# Use postgresql as the database for Active Record
gem 'pg', '~> 0.15'
# Use SCSS for stylesheets
gem 'sass-rails', '~> 5.0'
# Use Uglifier as compressor for JavaScript assets
gem 'uglifier', '>= 1.3.0'
# Use CoffeeScript for .coffee assets and views
gem 'coffee-rails', '~> 4.1.0'

# Use jquery as the JavaScript library
gem 'jquery-rails'
# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
gem 'turbolinks'
# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
gem 'jbuilder', '~> 2.0'
# bundle exec rake doc:rails generates the API under doc/api.
gem 'sdoc', '~> 0.4.0', group: :doc
 # Image storage for heroku production
gem 'aws-sdk', '< 2.0'
gem 'kaminari'
gem 'kaminari-i18n'
gem 'bootstrap-kaminari-views'
gem 'bootstrap-sass', '~> 3.3', '>= 3.3.6'
gem 'bootstrap-select-rails'

# gem 'datashift'
# gem 'datashift_spree'

gem 'solidus'
gem 'solidus_auth_devise'
gem 'solidus_i18n', '~> 1.0'
gem 'solidus_globalize', github: 'solidusio-contrib/solidus_globalize', branch: 'master'
# gem 'solidus_product_description_editor', '~> 1.0', '>= 1.0.1'
gem 'solidus_social', '~> 1.0'

gem 'puma'

group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug'
end

group :development do
  # Access an IRB console on exception pages or by using <%= console %> in views
  gem 'web-console', '~> 2.0'

  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
end

group :production do
  # Heroku helper
  gem 'rails_12factor'
 
end
```

**Comment by jasonfb** [comment]
see http://stackoverflow.com/questions/37244912/on-solidus-1-3-fresh-install-validation-failed-store-cant-be-blank-when-addin

**Comment by CBarklem** [comment]
Thank you.

Changing the default boolean to 'true' in spree stores table fixed the issue.

**Comment by jhawthorn** [comment]
@CBarklem as far as I can tell the seeds create a default=true store, and the sample data creates another non-default store. Is it possible you missed running `rake db:seed` (it should be part of the normal spree:install command)?

**Comment by jhawthorn** [comment]
There were indeed some issues with the "default" store in the database not being set as `default=true`. These should now be fixed in 1.3.0 by #1230
